### PR TITLE
[Components with more than 1 service] LPS-190421 Refactor ServerStatistics classes from portal-monitoring module

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-190421](https://liferay.atlassian.net/browse/LPS-190421)

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:startup

**Comment by RenanUmV** [comment]
ci:test:environments

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:environments

**Comment by Kvale1733** [comment]
ci:test:startup

File: modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portal/ServerStatistics.java [hunk 1]
```diff
@@ -24,12 +22,8 @@
  * @author Michael C. Han
  * @author Brian Wing Shun Chan
  */
-@Component(
-	enabled = false, property = "namespace=com.liferay.monitoring.Portal",
-	service = {DataSampleProcessor.class, ServerStatistics.class}
-)
-public class ServerStatistics
-	implements DataSampleProcessor<PortalRequestDataSample> {
+@Component(service = ServerStatistics.class)
```
Comment (rafaprax) [inline modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/portal/ServerStatistics.java]: Hey @RenanUmV, put the `enabled=false` property here as well. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:environments

**Comment by catarineaguiar** [comment]
ci:test:startup

**Review by rafaprax — APPROVED** [review]

**Comment by catarineaguiar** [comment]
no regressions found.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
ci:forward:force

cc @tinatian

**Comment by dantewang** [comment]
ci:forward:force

**Pull request closed without merging**